### ISSUE TITLE
Retrain and Keep Unchanged Gaussian points

### ISSUE THREAD

**Issue by Lk13Antkoo** [nerfstudio-project/gsplat#952]
Dear Gsplat team and everyone,

I am trying to train the  scene from a pretrained PLY file.
The pretrained PLY has the position (or the mean value) of each gaussian points. My purpose is training those points with the scene. 
I want to keep the position of of Gaussian points unchanged, I do not want to refine the gaussian (such as add or delete) and I just want to update the color parameters the covariances...

Can anyone let me know how to do that, what configuration I need to change?

Thank you and have a nice weekend,
AnPham

**Comment by egauh0113** [comment]
Have you tried setting `requires_grad=False` for "means"?
If you don't want to add or delete points, you could disable `_gros_gs()` and `_prune_gs()` in the strategy.
Without changing the stragtegy code, setting `refine_start_iter` to a large value may work.

**Comment by bell-one** [comment]
https://github.com/nerfstudio-project/gsplat/blob/c3d59bbb54f91d1c93f1a80dc86960c696b4a059/examples/simple_trainer.py#L295-L300

you can set learning rate in here for each params, 

maybe change means lr to zero could be helpful
